inc/saml/namespace.php

```diff
@@ -10,6 +10,8 @@
 use Altis;
 use HumanMade\SimpleSaml;
 
+use function Altis\get_environment_type;
```

Comment:
Reference the function using the namespace inline rather than using the function directly here.